loopchain/consensus/runner.py

```diff
@@ -54,6 +55,11 @@ def __init__(self,
             self.event_system, ChannelProperty().peer_address, self._block_factory, self._vote_factory
         )
 
+        self._loop = asyncio.get_event_loop()
+
+        self._is_broadcasting: bool = False
+        self._is_voting: bool = True
```

Comment:
Is there any reason it was set to True during initialization?

Reply:
:scream: :exclamation: 

Thank you